mage_ai/frontend/next.config.js

```diff
@@ -3,11 +3,13 @@
 const removeImports = require('next-remove-imports')();
 
 module.exports = removeImports({
+  output: 'export',
```

Comment:
Need to add `distDir` prop (see other [comment](https://github.com/mage-ai/mage-ai/pull/4168/files#r1424570927))
```
distDir: 'frontend_dist',
```

Reply:
I initially just tried using `../server/frontend_dist` for the `distDir`, but it was causing some issues running Mage locally on localhost:3000, and we also needed an easy way to export the frontend build to 2 separate directories (to handle base path configurations).

Reply:
added in 9489d5d0a